pkg/builder/builder.go

```diff
@@ -103,11 +103,27 @@ func BuildHlSvc(nm types.NamespacedName, vdb *vapi.VerticaDB) *corev1.Service {
 	}
 }
 
+// buildConfigVolumeMount returns the volume mount for config.
+// If vclusterops flag is enabled we mount only /opt/vertica/config/https_certs
+func buildConfigVolumeMount(vdb *vapi.VerticaDB) corev1.VolumeMount {
+	subPath := "config"
+	mountPath := paths.ConfigPath
+	if vmeta.UseVClusterOps(vdb.Annotations) {
+		subPath = "config/https_certs"
+		mountPath = paths.HTTPTLSConfDir
+	}
+	return corev1.VolumeMount{
+		Name:      vapi.LocalDataPVC,
+		SubPath:   vdb.GetPVSubPath(subPath),
+		MountPath: mountPath,
+	}
```

Comment:
I think it is clearer if you don't overwrite the variables. How about something like this?
```suggestion
	if vmeta.UseVClusterOps(vdb.Annotations) {
		return corev1.VolumeMount{
			Name:      vapi.LocalDataPVC,
			SubPath:   vdb.GetPVSubPath("config/https_certs"),
			MountPath: paths.HTTPTLSConfDir,
		}
	}
	return corev1.VolumeMount{
		Name:      vapi.LocalDataPVC,
		SubPath:   vdb.GetPVSubPath("config"),
		MountPath: paths.ConfigPath,
	}}
```